docs/concepts/crds/restoresession.md

```diff
@@ -131,6 +131,8 @@ A `RestoreSession` object has the following fields in the `spec` section.
 
 - **spec.target.volumeMounts :** `spec.target.volumeMounts` specifies a list of volumes and their `mountPath` where the data will be restored. Stash will mount these volumes inside the `init-container` or restore job.
 
+> **Note:** Stash take backup absolute path of a Volume. In RestoreSession, the `mountPath` must be same as the path of the original volume.
```

Comment:
> Note: Stash stores absolute path of the backed up files. Hence, your restored volume must be mounted on the same `mountPath` as the original volume. Otherwise, the backed up files will not be restored into your desired volume.